addons/project_issue_sheet/project_issue_sheet.py

```diff
@@ -19,15 +19,24 @@
 #
 ##############################################################################
 
+from openerp.addons.analytic.models import analytic
 from openerp.osv import fields,osv,orm
 from openerp.tools.translate import _
 
 class project_issue(osv.osv):
     _inherit = 'project.issue'
     _description = 'project issue'
+    
+    def _get_current_contract_state(self, cr, uid, ids, field_names, args, context=None):
+        res = {}
+        for issue in self.browse(cr, uid, ids, context=context):
+            res[issue.id] = issue.project_id and issue.project_id.partner_id and issue.project_id.use_issues and issue.project_id.analytic_account_id.state or False
```

Comment:
no need for `issue.project_id and issue.project_id.partner_id`, just do `issue.project_id.partner_id`